test/e2e/shim-install.sh

```diff
@@ -7,8 +7,8 @@ set -ex
 # Build gvisor-containerd-shim
 if [ "${INSTALL_LATEST}" == "1" ]; then
 { # Step 1(release): Install gvisor-containerd-shim
-LATEST_RELEASE=$(wget -qO - https://api.github.com/repos/google/gvisor-containerd-shim/releases | grep -oP '(?<="browser_download_url": ")https://[^"]*' | head -1)
-wget -O gvisor-containerd-shim
+LATEST_RELEASE=$(wget -qO - https://api.github.com/repos/google/gvisor-containerd-shim/releases | grep -oP '(?<="browser_download_url": ")https://[^"]*' | awk 'NR==2')
```

Comment:
I think this should be the following:
LATEST_RELEASE=$(wget -qO - https://api.github.com/repos/google/gvisor-containerd-shim/releases | grep -oP '(?<="browser_download_url": ")https://[^"]*gvisor-containerd-shim.linux-amd64' | head -1)

Reply:
@ianlewis done